Tests/SourceKitLSPTests/FoldingRangeTests.swift

```diff
@@ -303,4 +303,47 @@ final class FoldingRangeTests: XCTestCase {
       ]
     )
   }
+
+  func testFoldArgumentLabelsOnMultipleLines() async throws {
+    try await assertFoldingRanges(
+      markedSource: """
+        print(1️⃣
+          "x"
+        2️⃣)
+        """,
+      expectedRanges: [FoldingRangeSpec(from: "1️⃣", to: "2️⃣")]
+    )
+  }
+
+  func testFoldCallWithTrailingClosure() async throws {
```

Comment:
This one seems a little weird to me since we won't include a matching `)` after the fold. Should we just not provide 1->4 but rather 1->`)`?

Reply:
Yeah, maybe that’s the better solution. Also simplifies the implementation.